scripts/monitoring/auto_scale_workers.py

```diff
@@ -2,22 +2,27 @@
 import time
 import pytz
 import requests
-
-import boto3
-
-import botocore
+import warnings
 
 from auto_stop_workers import start_worker, stop_worker
+from prometheus_api_client import PrometheusConnect
+
+warnings.filterwarnings("ignore")
 
 utc = pytz.UTC
 
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+PROMETHEUS_URL = "https://monitoring-staging.eval.ai/prometheus/"
 
```

Comment:
Can we also read another environment variable which is named `ENV` with values `dev` or `prod`. If the environment is prod we should only perform auto scaling on the following challenge queues and ignore the rest:

1. VQA challenge: `vqa-challenge-2021-744-production-8228992d-b6c6-4e18-b6c4-ce2e8dbe9a57`
2. KILT challenge: `kilt-69d368bc-a5e8-4952-bd68-21c3ba61eb65`
3. NuScenes challenge: `nuscenes-detection-challenge-510c8c6d-a0d2-40bd-95dd-b7c8ea593d03`